jndi/jndi-url-itest/pom.xml

```diff
@@ -46,23 +46,24 @@
         <asm-debug-all.version>5.0.3</asm-debug-all.version>
         <cm.version>3.2.0-v20070116</cm.version>
         <depends-maven-plugin.version>1.5.0</depends-maven-plugin.version>
-        <geronimo-servlet_2.5_spec.version>1.2</geronimo-servlet_2.5_spec.version>
+        <geronimo-servlet_3.0_spec.version>1.0</geronimo-servlet_3.0_spec.version>
         <jndi-api.version>${jndi-api.dev-version}</jndi-api.version>
         <jndi-bundle.version>${jndi-bundle.dev-version}</jndi-bundle.version>
         <jndi-url-itest-biz.version>${jndi-url-itest-biz.dev-version}</jndi-url-itest-biz.version>
         <jndi-url-itest-web.version>${jndi-url-itest-web.dev-version}</jndi-url-itest-web.version>
         <jndi-url.version>${jndi-url.dev-version}</jndi-url.version>
         <logback.version>1.5.16</logback.version>
         <org.apache.aries.testsupport.unit.version>2.0.0-SNAPSHOT</org.apache.aries.testsupport.unit.version>
-        <org.apache.felix.fileinstall.version>3.4.0</org.apache.felix.fileinstall.version>
+        <org.apache.felix.fileinstall.version>3.7.4</org.apache.felix.fileinstall.version>
         <org.apache.servicemix.bundles.cglib.version>3.3.0_1</org.apache.servicemix.bundles.cglib.version>
-        <org.eclipse.osgi.version>3.11.3</org.eclipse.osgi.version>
-        <pax-exam.version>4.11.0</pax-exam.version>
-        <pax-url.version>2.6.0</pax-url.version>
-        <pax-web.version>1.1.18</pax-web.version>
-        <services.version>3.1.200-v20070605</services.version>
+        <org.eclipse.osgi.version>3.22.0</org.eclipse.osgi.version>
+        <pax-exam.version>4.13.0</pax-exam.version>
+        <pax-url.version>2.6.16</pax-url.version>
+        <pax-web.version>6.1.2</pax-web.version>
```

Comment:
version 7.x is not able to initialize one class from Jetty - skipping
version 8.x stucks on starting tomcat - https://github.com/apache/aries/pull/453
version 9.x does no support java 8